cmake/dnnl_compat.cmake

```
@@ -36,6 +36,7 @@ set(COMPAT_CACHE_BOOL_VARS
     "EXPERIMENTAL"
     "EXPERIMENTAL_UKERNEL"
     "EXPERIMENTAL_LOGGING"
+    "EXPERIMENTAL_SELF_SUFFICIENT_L0"
```

This name is a bit misleading, as Level Zero is not exposed anywhere in oneDNN API or build options. Consider something like:

```suggestion
    "EXPERIMENTAL_USE_SYCL_KERNEL_COMPILER
```